pyaedt/generic/filesystem.py

```diff
@@ -73,17 +73,21 @@ def remove(self):
 
     def copyfile(self, src_file, dst_filename=None):
         """
+        Copy a file to the scratch directory. The target filename is optional.
+        If omitted, the target file name is identical to the source file name.
 
         Parameters
         ----------
         src_file : str
             Source File with fullpath
         dst_filename : str, optional
-            Optional destination filename with extensione
+            Optional destination filename with extension.
```

Comment:
```suggestion
            Destination filename with the extension. The default is ``None``.
```
The optional is already noted after the data type.

Reply:
updated.